go/datas/remote_database_handlers.go

```diff
@@ -341,7 +341,7 @@ func handleRootGet(w http.ResponseWriter, req *http.Request, ps URLParams, rt ch
 	if req.Method != "GET" {
 		d.Panic("Expected get method.")
 	}
-
+	rt.Rebase()
```

Comment:
Ok, I took this out, but minimized the disturbance to the testing code by having `newHTTPChunkStoreForTest()` calling `Rebase()` each time it's about to forward a request to a handler function. It provides a good approximation of proper behavior, and Making It Right is tracked in #3415